Velaptor/Graphics/IRenderer.cs

```diff
@@ -131,11 +292,13 @@ public interface IRenderer
     /// <param name="text">The text to render.</param>
     /// <param name="x">The X coordinate location to render the text.</param>
     /// <param name="y">The Y coordinate location to render the text.</param>
```

Comment:
Refactor to a lower case "x" and "y", or refactor all of them to upper case "X" and "Y".
```suggestion
    /// <param name="x">The X coordinate location to render the text.</param>
    /// <param name="y">The Y coordinate location to render the text.</param>
```

Reply:
If you are talking about the `<param name="x">` and it being lowercase compared to the section `The X coordinate`, the XML reference `<param name="x">` HAS to be the lower case because it is a reference to the `int x` parameter of the method.  This is the way it is supposed to work.

Is this what you meant?

Reply:
@CalvinWilkinson 
No, I'm not talking about the param name. It's the text/documentation after the param name that needs to be consistent either upper or lowercase. I made the sections bold and italic.

`    /// <param name="x">The **_X coordinate_** location to render the text.</param>
    /// <param name="y">The **_Y coordinate_** location to render the text.</param>
`